modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/base_controller.rb

```diff
@@ -45,10 +45,12 @@ def request_representative
 
         def shared_form_validation(form_number)
           target_veteran
+          @user_profile = fetch_claimant
```

Comment:
This is only a suggestion, what do you think about refactoring this to something like:
```ruby
###### existing code
@claims_api_forms_validation_errors = validate_form_2122_and_2122a_submission_values(user_profile)
###### existing code
add_claimant_data_to_form if user_profile
########## existing code
def user_profile
  @user_profile ||= <your fetch_claimant method code>
end

def add_claimant_data_to_form
  if user_profile.status == :ok
    first_name = user_profile.profile.given_names.first
    last_name = user_profile.profile.family_name
    form_attributes['claimant'].merge!(firstName: first_name, lastName: last_name)
  end
end
```

Reply:
Nice, I'll update along those lines.